configure.ac

```diff
@@ -565,6 +565,59 @@ fi
 AM_CONDITIONAL([BUILD_WITH_KRB5], [test "$WITH_KRB5" != no])
 AC_SUBST([KRB5_LIB])
 
+
+
```

Comment:
Why are these here?  They are not bad, but it is not immediately obvious to me why this was added here.